src/main/java/frc/robot/subsystems/CanalSubsystem.java

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems;
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+/**
+ * Manages the canal subsystem, runs the front and back belts.
+ */
+public class CanalSubsystem extends SubsystemBase {
+  /** Creates a new CanalSubsystem. */
```

Comment:
is there supposed to be nothing in the canal subsystem?

Reply:
I don't know why, but the version of Canal Subsystem that actually has stuff on it was never on the repo. I'm pushing it now.